Mathlib/Tactic/Linter/Lint.lean

```diff
@@ -139,7 +139,7 @@ def missingEndLinter : Linter where run := withSetOptionIn fun stx ↦ do
     -- TODO: once mathlib's Lean version includes leanprover/lean4#4741, make this configurable
     unless #[`Mathlib, `test, `Archive, `Counterexamples].contains (← getMainModule).getRoot do
       return
-    if getLinterHash (← getOptions) && !(← MonadState.get).messages.hasErrors then
+    if getLinterMissingEnd (← getOptions) && !(← MonadState.get).messages.hasErrors then
```

Comment:
Should we just inline this?
```suggestion
    if Linter.getLinterValue linter.missingEnd (← getOptions) && !(← MonadState.get).messages.hasErrors then
```
Maybe we can even put `getLinterValue` in the namespace of `Lean.Option` so that we can write 
```suggestion
    if linter.missingEnd.getLinterValue (← getOptions) && !(← MonadState.get).messages.hasErrors then
```

Reply:
Do you really mean this change for just the `MissingEnd` linter, or for all of them?

Reply:
I mean inline all of them

Reply:
Done!  I did not introduce the alias, but I inlined them as is.